### ISSUE TITLE
Sticky comments above first import don't work

### ISSUE THREAD

**Issue by timgraham** [timothycrosley/isort#284]
A failing test:
```diff
diff --git a/test_isort.py b/test_isort.py
index d846749..1c200bf 100644
--- a/test_isort.py
+++ b/test_isort.py
@@ -1213,6 +1213,11 @@ def test_sticky_comments():
                   "from django.utils.translation import ugettext_lazy as _\n")
     assert SortImports(file_contents=test_input).output == test_input
 
+    test_input = ("# Comment one\n"
+                  "from django import forms\n"
+                  "# Comment two\n"
+                  "from django.contrib.gis.geos import GEOSException\n")
+    assert SortImports(file_contents=test_input).output == test_input
 
 
 def test_zipimport():
```
"Comment one" will be sorted below `from django import forms`.

**Comment by timothycrosley** [comment]
@timgraham thanks for reporting this issue!

I've verified it and plan to have a fix in before the next release.

**Comment by timothycrosley** [comment]
Fixed in version 4.1.0

Thanks!

~Timothy
